### PR TITLE
[frontend] Manually fix Lint/DuplicateMethods offense

### PR DESCRIPTION
**Pull request by bgeuken**
This fixes offenses caused by duplicated method definitions. In the
node.rb file we drop the 'attr_reader :transport', since this got
overwritten by a later definition of the transport method.

Mob-programming with @DavidKang

### CODE REVIEW

**Comment by bgeuken** [comment]
@openSUSE/open-build-service Please review

File: src/api/lib/activexml/node.rb [hunk 1]
```diff
@@ -40,10 +40,7 @@ def reset_runtime
         @@xml_time = 0
       end
 
-      # transport object, gets defined according to configuration when Base is subclassed
-      attr_reader :transport
```
Comment (Ana06) [inline src/api/lib/activexml/node.rb]: As I already commented in the other PR:

are you sure that you can remove this? I know that there is also a `transport` method in this class, but if I am not wrong, without this you can not access the `@@transport` variable from outside anymore. And as it not used here neither... why do we have the `@@transport` object variable? 🤔

File: src/api/lib/activexml/node.rb [hunk 2]
```diff
@@ -40,10 +40,7 @@ def reset_runtime
...
-
-      def inherited( subclass )
+      def inherited(subclass)
```
Comment (Ana06) [inline src/api/lib/activexml/node.rb]: unrelated change!
